packages/yoastseo/README.md

```diff
@@ -78,6 +77,9 @@ worker.initialize( {
 } );
 ```
 
+There is a basic example in the `apps/content-analysis-webworker` folder, which also contains a basic setup with Webpack.
+There is also a more involved example in the `apps/content-analysis` folder, which has a basic React implementation.
+
```

Comment:
These paths `apps/content-analysis-webworker` and `apps/content-analysis` might be redundant for a user that accesses `yoastseo` from npm for example. Maybe we can use the link instead, like `https://github.com/Yoast/wordpress-seo/tree/trunk/apps/content-analysis-api`? 🤔